tests/framework/ROM/SKLearn/gold/data/outLinearRegressionModel.xml

```diff
@@ -1,9 +1,10 @@
 <DataObjectMetadata name="modelOut">
   <modelUnderTest type="Static">
     <ROM>
-      <type>SupervisedLearning</type>
-      <coef_Z>-0.0426499542795,-5.43142021766e-05</coef_Z>
-      <intercept_Z>0.275851025262</intercept_Z>
```

Comment:
These should not be removed (since we are testing to make sure that the coefficients are being exported)

Reply:
reverted (now it also adds the "Features and Targets")